wxwidgets.yaml

```diff
@@ -0,0 +1,33 @@
+name: wxwidgets
+rm-configure: true
+config-opts:
+  - --enable-shared
+  - --disable-static
+  - --enable-unicode
+  - --enable-display
+  - --enable-propgrid
+  - --enable-stc
+  - --with-gtk=3
+  - --with-opengl
+  - --with-libjpeg
+  - --with-libpng
+  - --with-zlib
+  - --disable-webkit
+  - --disable-webview
+  - --disable-ribbon
+sources:
+  - type: archive
+    url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2
+    sha256: 96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
+  - type: shell
+    commands:
+      - cp -p /usr/share/automake-*/config.{sub,guess} .;
+      - autoconf -f -B "build/autoconf_prepend-include";
+post-install:
+  - install -p -D -m 0644  "docs/gpl.txt" "docs/lgpl.txt" "docs/licence.txt" -t "${FLATPAK_DEST}/share/licenses/wxwidgets/";
```

Comment:
The files isn't exported, it's part of Sources extension.

Reply:
Done.

https://github.com/flathub/flathub/pull/801/commits/de93f9e911a070beb1913aade48a4b7f544a2eb9#diff-3ddf6d991962b672e6681373d69e0cf1L27